kano_updater/ui/progress.py

```diff
@@ -24,10 +24,12 @@ def _change(self, phase, msg):
                       phase.get_main_phase().label, phase.get_main_phase().name, msg)
 
     def _error(self, phase, msg):
+        kill_flappy_judoka()
```

Comment:
Don't you already do this in the `self._window.error()` function. Regardless, this progress object shouldn't contain any information about the details of the GUI - it should only be used to connect the backend with the GUI.